src/audio-worklet/migration/spn-recorder/app.js

```diff
@@ -147,17 +156,25 @@ function setupScriptProcessor(recordingProperties, passSampleToVisualizers) {
  */
 function setupRecording(recordingProperties) {
   recordButton.addEventListener('click', (event) => {
-    isRecording = !isRecording;
+    isRecording = true;
+    changeButtonStatus();
+  });
```

Comment:
If you're planning to make this as an one-time event, let's add `{once: true}` here as well.

Reply:
I don't think it is a one-time event. Based on my understanding of SPN, we use "IsRecording" boolean to control whether we need to write element into buffer and then use "setupRecording" to hear whether there is click event in the entire recording process. Therefore, we need it to be call multiple times, in case, user want to start and stop several times

Reply:
Okay, I would love to upgrade it, therefore both file can look consistent.